draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2208,7 +2208,7 @@ specified to be used (SCT, STH, or both).
 
 ### Log ID Registry {#log_id_registry}
 
-IANA is asked to establish a registry of Log IDs, named "Log ID Registry",
+IANA is asked to establish a registry of Log IDs, named "Log ID",
```

Comment:
Nit: Paul suggested "Log IDs" in the plural.

Reply:
Updated commit pushed.